src/lib/components/ScrollToTopButton.svelte

```diff
@@ -0,0 +1,105 @@
+<script lang="ts">
+  export let scrollYValue;
+  import { FontAwesomeIcon } from 'fontawesome-svelte';
+  import { fade } from 'svelte/transition';
+</script>
+
+{#if scrollYValue >= 1000}
+  <div class="back-to-top-wrapper" in:fade={{duration: 200}} out:fade={{duration: 200}}>
+    <a href="#top" class="back-to-top-link" aria-label="Scroll to Top">
+      <FontAwesomeIcon icon={['fas', 'arrow-up']}/>
+    </a>
+  </div>
+{/if}
+<style lang="scss">
```

Comment:
I'm using SASS in all other components, can you convert it?

I think you'll only have to do the following:
- remove semicolons
- remove curly braces
- keep whitespace indention